public/home.json

```diff
@@ -79,7 +79,7 @@
         "description": "Join us for a discussion of this essential guide to writing clear and maintainable code!\n",
         "images": [
           {
-            "path": "https://cloudprovider.com/image.png",
+            "path": "/event-mobile-img.png",
```

Comment:
Considering we are going to use later the same structure, what about the create a folder events and move the image there?

Reply:
Hey @dricazenck 
I'm going to work on the overall event component that holds all the even cards. Can I update it when I do? Let me know.

Reply:
all good :)